src/atlas/meshgenerator/detail/StructuredMeshGenerator.cc

```diff
@@ -65,6 +65,10 @@ struct Region {
 };
 
 StructuredMeshGenerator::StructuredMeshGenerator( const eckit::Parametrisation& p ) {
+
+    Log::info() << "StructuredMeshGenerator::config parameterisation" <<
+       dynamic_cast<const eckit::Configuration &>(p) << std::endl;
+
```

Comment:
Please remove this statement, it is too verbose for many applications.

Reply:
Done!